pcsx2/GS/Window/GSwxDialog.cpp

```diff
@@ -0,0 +1,755 @@
+/*
```

Comment:
Inconsistent copyright header, should be
https://github.com/PCSX2/pcsx2/blob/master/pcsx2/GS/GS.cpp